ai-gateway/guides/troubleshooting.mdx

```diff
@@ -32,7 +32,7 @@ The request could not be proxied to the AI provider successfully
 
 **Message**
 
-The request's body could not be parsed: {err}
+Invalid request body: {err}
```

Comment:
<img width="313" height="84" alt="Screenshot 2025-12-15 at 1 04 35 PM" src="https://github.com/user-attachments/assets/96226af3-2adf-438e-80cc-0d7db46a9140" />

just noticed in the deployment that the "{err}" doesn't render